larky/src/main/java/com/verygood/security/larky/modules/vgs/metrics/MetricsModule.java

```diff
@@ -0,0 +1,68 @@
+package com.verygood.security.larky.modules.vgs.metrics;
+
+import com.google.common.collect.ImmutableList;
+import com.verygood.security.larky.modules.vgs.metrics.spi.LarkyMetrics;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.starlark.java.annot.Param;
+import net.starlark.java.annot.ParamType;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.Dict;
+import net.starlark.java.eval.EvalException;
+
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+@Getter
+@Slf4j
+@StarlarkBuiltin(
+  name = "metrics",
+  category = "BUILTIN",
+  doc = "Overridable Metrics API in Larky")
+public class MetricsModule implements LarkyMetrics {
+
+  public static final MetricsModule INSTANCE = new MetricsModule();
+  public static final String ENABLE_DEFAULT_PROPERTY = "larky.modules.vgs.metrics.enableDefaultMetrics";
+
+  private final LarkyMetrics metrics;
+
+  public MetricsModule() {
+    ServiceLoader<LarkyMetrics> loader = ServiceLoader.load(LarkyMetrics.class);
+    List<LarkyMetrics> metricsProviders = ImmutableList.copyOf(loader.iterator());
+
+    if (Boolean.getBoolean(ENABLE_DEFAULT_PROPERTY)) {
```

Comment:
what is "default metrics"? 🤔

Reply:
It is just used for testing in starlarky, not used in production

Reply:
Why does it need a property?

Reply:
removed `ENABLE_DEFAULT_PROPERTY`